GETTING_STARTED.md

```diff
@@ -584,6 +585,28 @@ factory :post do
 end
 ```
 
+### Association overrides
+
+Attribute overrides can be used to link associated objects:
+
+```ruby
```

Comment:
Let's ditch the comments and then leave out the attributes that aren't relevant to this section. I would also like to switch to the implicit style of declaring the `author` association, which works in more cases than the inline style.

In my first set of comments I was thinking we would leave off the factory definition entirely, since we sort of cover it in previous sections, but now seeing it I like that you kept it. But I think we can get it a bit more focused.

With my comments I think we will end up with:

```ruby
FactoryBot.define do
  factory :author

  factory :post do
    author
   end
end

author = build(:author)
post = build(:post, author: author)
```

Reply:
Apologies for taking so long to get back to this! I applied your changes and pushed a new commit.

Reply:
No worries! Thanks for taking the time to improve the documentation!